install.md

```diff
@@ -37,13 +37,28 @@ cd conmon
 make
 ```
 
+## Install with Ansible
+
+An [Ansible Role](https://github.com/alvistack/ansible-role-conmon) is also available to automate the above steps:
+
+``` bash
+sudo su -
+mkdir -p ~/.ansible/roles
+git clone https://github.com/alvistack/ansible-role-conmon.git conmon
+cd ~/.ansible/roles/conmon
+pip3 install --upgrade --ignore-installed --requirement requirements.txt
+molecule converge
+molecule verify
+```
 
 ## Install to the directory you want:
+
 There are three options for installation, depending on your environment. Each can have the PREFIX overridden. The PREFIX defaults to `/usr/local` for most Linux distributions.
 
-- `make install` installs to `$PREFIX/bin`, for adding conmon to the path.
-- `make podman` installs to `$PREFIX/libexec/podman`, which is used to override the conmon version that Podman uses.
-- `make crio` installs to `$PREFIX/libexec/crio`, which is used to override the conmon version that CRI-O uses.
+  - `make install` installs to `$PREFIX/bin`, for adding conmon to the path.
+  - `make podman` installs to `$PREFIX/libexec/podman`, which is used to override the conmon version that Podman uses.
+  - `make crio` installs to `$PREFIX/libexec/crio`, which is used to override the conmon version that CRI-O uses.
 
+<!-- end list -->
```

Comment:
The same applies for those and the comment.

Reply:
It is done by ‘pandoc -t gfm -f gfm --wrap preserve -o install.md install.md’

Reply:
rebase done